ydb/core/kqp/executer_actor/kqp_executer_stats.cpp

```diff
@@ -64,26 +66,32 @@ void TStageExecutionStats::Resize(ui32 taskCount) {
     MaxMemoryUsage.resize(taskCount);
 }
 
+void SetNonZero(ui64& target, ui64 source) {
+    if (source) {
+        target = source;
+    }
+}
+
 void TStageExecutionStats::UpdateAsyncStats(i32 index, TAsyncStats& aggrAsyncStats, const NYql::NDqProto::TDqAsyncBufferStats& asyncStats) {
-    aggrAsyncStats.Bytes[index] = asyncStats.GetBytes();
-    aggrAsyncStats.Rows[index] = asyncStats.GetRows();
-    aggrAsyncStats.Chunks[index] = asyncStats.GetChunks();
-    aggrAsyncStats.Splits[index] = asyncStats.GetSplits();
+    SetNonZero(aggrAsyncStats.Bytes[index], asyncStats.GetBytes());
```

Comment:
That's OK since:

1. I excplicitly call resize() for every std::vector() and zero is correct default
2. The main problem occurs when 0 overrides non-zero value. It should never occurs since all metrics have to grow monotonically. But is metric is missed for some reason, it is arrived as zero in protobuf abd we have to ignore it